<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lab10_oopClass</class>
 <widget class="QMainWindow" name="lab10_oopClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lab10_oop</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>mode A</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>2</y>
        <width>591</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="plantsListWidget">
         <property name="font">
          <font>
           <family>Thames</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Coded Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="codedNameLineEdit">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Species</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="speciesLineEdit">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Age in Months</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="ageInMonthsLineEdit">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Digitized Scan</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="digitizedScanLineEdit">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="addButton">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="deleteButton">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="updateButton">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="exitButton">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="undo_pushButton">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Undo</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="redo_pushButton">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Redo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>mode B</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>2</y>
        <width>591</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QListWidget" name="mylistListWidget">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="filteredListWidget">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Coded Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="codedNameAsisstantlineEdit">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Species</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="speciesAsisstantlineEdit">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Age in Months</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="ageInMonthsAsisstantlineEdit">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Digitized Scan</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="digitizedScanAsisstantlineEdit">
           <property name="font">
            <font>
             <family>Thames</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="saveButton">
             <property name="font">
              <font>
               <family>Thames</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="filterButton">
             <property name="font">
              <font>
               <family>Thames</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="openButton">
             <property name="font">
              <font>
               <family>Thames</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Open</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="nextButton">
             <property name="font">
              <font>
               <family>Thames</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Next</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mylist_pushButton">
             <property name="font">
              <font>
               <family>Thames</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>My List</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="exitAsisstantButton">
             <property name="font">
              <font>
               <family>Thames</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Exit</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlant_Clusters">
    <property name="title">
     <string>Plant Clusters</string>
    </property>
   </widget>
   <addaction name="menuPlant_Clusters"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="lab10_oop.qrc"/>
 </resources>
 <connections/>
</ui>
